examples/etl/basic_example.py

```diff
@@ -2,51 +2,54 @@
 from pyspark.sql import DataFrame
 from pyspark.sql.types import StructType, StructField, IntegerType, StringType
 
-from atc.etl import Extractor, Transformer, Loader, Orchestration
+from atc.etl import Extractor, Transformer, Loader, Orchestrator
 from atc.spark import Spark
 
 
 class GuitarExtractor(Extractor):
     def read(self) -> DataFrame:
         return Spark.get().createDataFrame(
-            Spark.get().sparkContext.parallelize([
-                ('1', 'Fender', 'Telecaster', '1950'),
-                ('2', 'Gibson', 'Les Paul', '1959'),
-                ('3', 'Ibanez', 'RG', '1987')
-            ]),
-            StructType([
-                StructField('id', StringType()),
-                StructField('brand', StringType()),
-                StructField('model', StringType()),
-                StructField('year', StringType()),
-            ]))
+            Spark.get().sparkContext.parallelize(
+                [
+                    ("1", "Fender", "Telecaster", "1950"),
+                    ("2", "Gibson", "Les Paul", "1959"),
+                    ("3", "Ibanez", "RG", "1987"),
+                ]
+            ),
+            """
+            id STRING,
+            brand STRING,
+            model STRING,
+            year STRING
+            """,
+        )
 
 
 class BasicTransformer(Transformer):
     def process(self, df: DataFrame) -> DataFrame:
-        print('Current DataFrame schema')
+        print("Current DataFrame schema")
         df.printSchema()
 
-        df = df.withColumn('id', f.col('id').cast(IntegerType()))
-        df = df.withColumn('year', f.col('year').cast(IntegerType()))
+        df = df.withColumn("id", f.col("id").cast(IntegerType()))
+        df = df.withColumn("year", f.col("year").cast(IntegerType()))
 
-        print('New DataFrame schema')
+        print("New DataFrame schema")
         df.printSchema()
         return df
 
 
 class NoopLoader(Loader):
-    def save(self, df: DataFrame) -> DataFrame:
-        df.write.format('noop').mode('overwrite').save()
-        return df
+    def save(self, df: DataFrame) -> None:
+        df.write.format("noop").mode("overwrite").save()
```

Comment:
why not returning the df?

Reply:
https://github.com/atc-net/atc-dataplatform/pull/57#discussion_r780823779